packages/vue-i18n-routing/package.json

```diff
@@ -39,10 +39,10 @@
   },
   "peerDependencies": {
     "@vue/composition-api": "^1.4.0",
-    "vue": "^2.6.14 || ^3.2.0",
+    "vue": "^2.6.14 || ^2.7.0 || ^3.2.0",
```

Comment:
Does it make sense to have both `^2.6.14` and `^2.7.0`? The former already takes care of the latter and what does the latter really mean in presence of the former? Does it makes the whole restricted to `^2.7.0` or not?

Same for the `vue_router` change below.

Reply:
I can see in https://semver.npmjs.com/ that `^2.6.14 || ^2.7.0` matches from `^2.6.14` so that's good but then I don't see much point in adding `^2.7.0` anyway.

Reply:
Thank you for your comment.
The semver spec certainly doesn't make sense, but I don't know why, but it didn't work with `peerDependencies`, so I did 😅